engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2402,6 +2401,7 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
           LOG.warn("Cannot flush command history");
         }
       }
+      yarnClient.stop();
```

Comment:
The javadoc seems to indicate stop is the method to the call and close is to provide the closeable support with an explicit message in close javadoc that says "Implementation classes MUST relay this directly to {@link #stop()}"